scripts/scil_resample_volume.py

```diff
@@ -71,46 +66,22 @@ def main():
     logging.debug('Loading Raw data from %s', args.input)
 
     img = nib.load(args.input)
-    data = img.get_data()
-    affine = img.get_affine()
-    original_zooms = img.get_header().get_zooms()[:3]
 
+    # Resampling volume
     if args.ref:
-        ref_img = nib.load(args.ref)
-        new_zooms = ref_img.header.get_zooms()[:3]
+        resampled_image = resample_volume(img, ref=args.ref,
```

Comment:
You could just call it once with all parameters, which should be None if they were not provided